views/games.pug

```diff
@@ -4,10 +4,11 @@ append head
     link(rel="stylesheet" href="/stylesheets/allGames.css")
     script(src="/javascripts/index.js" type="module" defer)
 block content
-    div.allGameBox
-        h1.allGamesHeader All Games
-        div.games
-            each game in games
-                div.game
-                    a.games(href=`/games/${game.id}/reviews`)
-                        img(src=`${game.imageSrc}` class="gameThumbnail")
+    body
```

Comment:
I don't think this is needed, the body tag already exists in the layout.pug that is extended.

Reply:
Body tag allowed me to easily style the rest of the page in line with rest of site aesthetic